test/integration/test_azure_stt_live.py

```
@@ -0,0 +1,117 @@
+    def register(app, model):
+        app.include_router(make_azure_stt_router(model))
+
+    yield from run_live_server(register)
+
+
+# ---------------------------------------------------------------------------
+# REST
+# ---------------------------------------------------------------------------
+
+def test_rest_short_form(base_url):
+    r = requests.post(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)